wcfsetup/install/files/lib/data/user/UserAction.class.php

```diff
@@ -670,7 +726,8 @@ public function disable() {
 		
 		$action = new UserAction($this->objects, 'update', [
 			'data' => [
-				'activationCode' => UserRegistrationUtil::getActivationCode()
+				'activationCode' => UserRegistrationUtil::getActivationCode(),
+				'emailConfirmed' => bin2hex(\random_bytes(20)),
```

Comment:
I feel that unconfirming the email address should not be part of `disable`, because there is a dedicated method for this.

Reply:
The problem here is that if I don't reset the email confirmation when disabling, the behavior is undefined. If users can unlock themselves through an e-mail confirmation, there can, theoretically, not be the status: Disables but valid e-mail. Therefore, I would leave the behavior as it is.

Reply:
Okay, fair enough. Add a comment about this then.